openedx/core/djangoapps/content/course_overviews/tests.py

```diff
@@ -644,6 +645,32 @@ def test_disabled_with_prior_data(self, modulestore_type):
         )
 
     @ddt.data(ModuleStoreEnum.Type.mongo, ModuleStoreEnum.Type.split)
+    def test_cdn(self, modulestore_type):
+        """
+        Test that we return CDN prefixed URLs if it is enabled.
+        """
+        with self.store.default_store(modulestore_type):
+            course = CourseFactory.create(default_store=modulestore_type)
+            overview = CourseOverview.get_from_id(course.id)
+
+            # First the behavior when there's no CDN enabled...
+            AssetBaseUrlConfig.objects.all().delete()
+            if modulestore_type == ModuleStoreEnum.Type.mongo:
+                expected_path_start = "/c4x/"
+            elif modulestore_type == ModuleStoreEnum.Type.split:
+                expected_path_start = "/asset-v1:"
+
+            for url in overview.image_urls.values():
+                self.assertTrue(url.startswith(expected_path_start))
+
+            # Now enable the CDN...
+            AssetBaseUrlConfig.objects.create(enabled=True, base_url='fakecdn.edx.org')
```

Comment:
This will be based on a django setting, no? What happens if someone put in a trailing slash: `fakecdn.edx.org/`? [perhaps the url parsing is smart enough?]